Apps/phwindowsdefenderatp/windowsdefenderatp.json

```diff
@@ -48,22 +48,21 @@
     "actions": [
         {
             "action": "test connectivity",
-            "identifier": "test_connectivity",
             "description": "Validate the asset configuration for connectivity using the supplied configuration",
             "type": "test",
+            "identifier": "test_connectivity",
             "read_only": true,
             "parameters": {},
             "output": [],
             "versions": "EQ(*)"
         },
         {
             "action": "quarantine device",
-            "identifier": "quarantine_device",
             "description": "Quarantine the device",
             "verbose": "This action can take a while to complete execution. The action retries at an interval of 5 seconds within a specified <b>timeout</b> to check if event execution is complete. If not, the action would return the latest status of the event along with event ID.<br>The <b>get status</b> action can be used to fetch the latest status of the event with the event ID.<br>For parameter <i>type</i>, <b>&quotFull&quot</b> will completely quarantine the device while <b>&quotSelective&quot</b> will allow Skype and Outlook to be accessed.<br>The maximum timeout period for a status check is 10 minutes.",
             "type": "contain",
+            "identifier": "quarantine_device",
             "read_only": false,
-            "undo": "unquarantine device",
```

Comment:
why did we remove the "undo" key? I didn't see that as part of the original contributor's submission

Reply:
Usually, we keep the undo option for those actions which have the same number of action parameters. Because if we don't keep as that way then when we run the undo action there will be chances that some value will be lost or the user has to add some values due to a difference in the number of action parameters. Do we expect anything else for the 'undo' option?